pyproject.toml

```diff
@@ -3,11 +3,12 @@ requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
 [project]
+requires-python = ">= 3.6"
```

Comment:
I wonder if this is not too strict?

Reply:
This was copied from the old classifiers

```toml
classifiers = [
    'Development Status :: 5 - Production/Stable',
    'Intended Audience :: Developers',
    'Topic :: Software Development :: Build Tools',
    'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
    'Programming Language :: Python :: 3.6',   # slc7, slc8, cs8
    'Programming Language :: Python :: 3.8',   # ubuntu2004
    'Programming Language :: Python :: 3.9',   # slc9
    'Programming Language :: Python :: 3.10',  # ubuntu2204
    'Programming Language :: Python :: 3.11',  # MacOS
    'Programming Language :: Python :: 3.12',  # MacOS
    'Programming Language :: Python :: 3.13',  # MacOS
]
```